sdcm/sct_events/system.py

```diff
@@ -222,3 +223,22 @@ def __reduce__(self):
         """Need to define it for pickling because of Exception class in MRO."""
 
         return type(self), (self.test_status, self.events, self.event_timestamp)
+
+
+class InstanceStatusEvent(LogEvent, abstract=True):
+    STARTUP: Type[LogEventProtocol]
+    REBOOT: Type[LogEventProtocol]
+    POWER_OFF: Type[LogEventProtocol]
+
+
+InstanceStatusEvent.add_subevent_type("STARTUP", severity=Severity.WARNING, regex="kernel: Linux version")
+InstanceStatusEvent.add_subevent_type("REBOOT", severity=Severity.WARNING,
+                                      regex="Stopped target Host and Network Name Lookups")
+InstanceStatusEvent.add_subevent_type("POWER_OFF", severity=Severity.WARNING, regex="Reached target Power-Off")
+
+INSTANCE_STATUS_EVENTS = (InstanceStatusEvent.STARTUP(),
+                          InstanceStatusEvent.REBOOT(),
+                          InstanceStatusEvent.POWER_OFF())
```

Comment:
Please, use same style for the tuple items as in other files (see `database.py` for example)

Reply:
changed